zc_install/sql/updates/mysql_upgrade_zencart_158.sql

```diff
@@ -224,6 +224,10 @@ UPDATE configuration SET val_function = '{"error":"TEXT_EMAIL_ADDRESS_VALIDATE",
 ### Added v158 bring banners_titles up to date
 UPDATE banners SET banners_title = 'Zen Cart Certified Services' WHERE banners_html_text LIKE '%<script>%';
 
+#############
+### Added to correct storpicku tax basis
```

Comment:
please correct typo

Reply:
Have corrected. storpicku to storepickup.